packages/xeus/src/coincident.worker.ts

```diff
@@ -84,3 +97,8 @@ workerAPI.ready = worker.ready.bind(worker);
 workerAPI.cd = worker.cd.bind(worker);
 workerAPI.isDir = worker.isDir.bind(worker);
 workerAPI.processMessage = worker.processMessage.bind(worker);
+
+// @ts-expect-error quick and dirty way to add coincident support
+workerAPI.storeAsGlobal = worker.storeAsGlobal.bind(worker);
+// @ts-expect-error quick and dirty way to add coincident support
+workerAPI.callGlobalReciver = worker.callGlobalReciver.bind(worker);
```

Comment:
Can you extend the interface https://github.com/jupyterlite/xeus/blob/c845708d901630762969b669ad15b09bd274c89f/packages/xeus-core/src/interfaces.ts#L21

Defining `storeAsGlobal` and `callGlobalReceiver` there will help us remove the ts-expect-error calls